flocker/node/_config.py

```diff
@@ -1289,14 +1307,14 @@ def current_from_configuration(current_configuration):
     return Deployment(nodes=frozenset(nodes))
 
 
-def marshal_configuration(state):
+def marshal_node_state(state):
```

Comment:
If you're convinced by any of the `from_config`-style suggestions above then a `to_config`-style name might make more sense here.  `node_state_to_config`?  I'm not 100% behind this idea though, since we seem to be trying to draw a distinction between "desired config" and "deployment state" even though in practice the representation of these two things is pretty similar.  Since this is deployment state, not desired config, perhaps putting "config" in the name is confusing (and the parallel idea, "to_deployment_state" doesn't really make sense).